github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
No, the use of SQLite is very localized.

Sorry about the huge monkey-patch for caching. I couldn't find any other way to inject the cacher; the github_api ought to support it but it has been broken over a year.